index.js

```diff
@@ -99,7 +103,10 @@ function resolveUrlLoader(content, sourceMap) {
   if (useMap) {
 
     // source-map sources seem to be relative to the file being processed
-    absoluteToRelative(reworked.map.sources, filePath);
+    absoluteToRelative(reworked.map.sources, path.resolve(filePath, sourceRoot));
```

Comment:
The paths in the generated source map were not correct. So basically the paths were resolved relative to a wrong directory. 

INCLUDING this change the sources look like this (correct):
```json
{
"sources": [
    "webpack:///../~/open-sans-fontface/sass/_Regular.scss",
    "webpack:///../~/open-sans-fontface/sass/_mixins.scss",
    "webpack:///./components/Test.scss",
    "webpack:///../~/open-sans-fontface/sass/_Bold.scss",
    "webpack:///./styles/styles.scss",
    "webpack:///../~/compass-mixins/lib/compass/css3/_shared.scss",
    "webpack:///./components/sub/Test2.scss"
  ]
}
```

Without the paths are not resolved ocrrectly in the final source map:

```json
{
  "sources": [
    "webpack:///C:/Users/xxx/Software/opensource/~/open-sans-fontface/sass/_Regular.scss",
    "webpack:///C:/Users/xxx/Software/opensource/~/open-sans-fontface/sass/_mixins.scss",
    "webpack:///./components/Test.scss",
    "webpack:///C:/Users/xxx/Software/opensource/~/open-sans-fontface/sass/_Bold.scss",
    "webpack:///../styles/styles.scss",
    "webpack:///./Test.scss",
    "webpack:///C:/Users/xxx/Software/opensource/~/compass-mixins/lib/compass/css3/_shared.scss",
    "webpack:///./components/sub/Test2.scss",
    "webpack:///./components/Test2.scss"
  ]
}
```